upcase_booklist_app/upcase_booklist_app.py

```diff
@@ -29,5 +31,28 @@ def authors():
     return render_template("authors.html", books=books)
 
 
+@app.route("/lists")
+def lists():
```

Comment:
I'm slightly confused on the verbiage between lists and categories. It would seem they are the same thing...so you would you the singular and plural forms of one, such as `lists` and `list`.